test/libsolidity/semanticTests/abiEncoderV2/calldata_nested_array_reencode.sol

```diff
@@ -0,0 +1,14 @@
+pragma abicoder v2;
+
+contract C {
+        function h(uint[][] calldata a) public {
```

Comment:
Can you add a case with a 1D array as well? Looking through the test dir I don't see a case covering that.

Reply:
One-dimensional arrays are immediately validated during the initial abi decoding, so that'll revert even for an empty function body - I'd think (hope :-)) there's some tests for that somewhere, but I'll add another just to be sure.